components/appointmentPicker/appointmentPickerController.js

```diff
@@ -5,7 +5,7 @@ define('appointmentPickerController', [], function() {
 		$scope.sites = [];
 		$scope.dates = [];
 		$scope.today = new Date();
-		$scope.sharedProperties = sharedPropertiesService.getSharedProperties();
+		$scope.appointmentPickerSharedProperties = sharedPropertiesService.getSharedProperties();
```

Comment:
@CollinsSpencer I had to change the name of this because if they were both named `sharedProperties`, then they would overwrite each other. I don't fully understand scoping in Angular, so the easiest solution was to simply give this a more specific name that wouldn't conflict.